soe/src/sites/default/settings.php

```diff
@@ -1,15 +1,23 @@
 <?php
 
+//Pull Acquia Global environments.
+if (file_exists('/var/www/site-php')) {
+  require '/var/www/site-php/swssoe/swssoe-settings.inc';
+}
+
 // Require HTTPS.
 // We're behind a load-balancer, so we can't check $_SERVER['HTTPS'].
 // Have to check HTTP_X_FORWARDED_PROTO.
 // $_SERVER['HTTP_X_FORWARDED_PROTO'] is only set when we're serving over https,
 // therefore check if it's set.
-if (!array_key_exists('HTTP_X_FORWARDED_PROTO', $_SERVER) && !drupal_is_cli()) {
-  header('HTTP/1.0 301 Moved Permanently');
-  $redirect_url = "https://" . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'];
-  header("Location: " . check_plain($redirect_url));
-  exit();
+// Also checks for Acquia variables to determine if site is a local site or development.
+if(isset($_ENV['AH_SITE_ENVIRONMENT'])){
```

Comment:
Why not:
```
if(isset($_ENV['AH_SITE_ENVIRONMENT']) && !array_key_exists('HTTP_X_FORWARDED_PROTO', $_SERVER) && !drupal_is_cli())
```
(Not sure if my parentheses are correct, but you get the gist.)

Reply:
Definitely a gut call. While I try to avoid nesting conditionals, I feel like I've had experiences where things got squirrelly with too many logical operators.

Reply:
I did a little research and from what I can tell, there is no reason why your edit won't work fine.

Reply:
Yeah, agreed that the nested is more readable. Either way, please add a code comment for the `HTTP_X_FORWARDED_PROTO` and `drupal_is_cli()` checks.

Reply:
I don't have a strong preference.

Reply:
@jbickar Okay. I'll let it be then. 

I pushed a change with the comment.